### ISSUE TITLE
Callsigns with / show up as new contacts (highlighted red) even though they correctly show previous QSOs

### ISSUE THREAD

**Issue by ryandzink** [magicbug/Cloudlog#557]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'QSO -> Post QSO'
2. Enter details for a QSO with a station that has a / in it (in my case K8YSE/7)
3. Previous contacts with that station are correctly shown on the sidebar
4. The callsign field is highlighted red and indicates this is a new callsign

**Expected behaviour**
The callsign field should be green and contact details for the callsign should auto populate. I have only seen this happen on callsigns with slashes in them so I am guessing it is something to do with character escaping? This particular contact is happening for the same satellite/bands on which I have made contact in the past
![Capture](https://user-images.githubusercontent.com/13768946/86688825-4b386c80-bfbb-11ea-9f3e-b1469f35f33f.PNG)
.

**Screenshots**
1 screenshot attached

**Comment by ryandzink** [comment]
I don't think this is isolated to callsigns with slashes since I just experienced the same with a callsign with no slashes. Am I just not understanding the criteria that would cause the callsign to be considered a new callsign?
![Capture-2](https://user-images.githubusercontent.com/13768946/86693108-3c53b900-bfbf-11ea-8149-806427775d67.PNG)

**Comment by magicbug** [comment]
With the above screenshot with FM/2M is the Satellite tab set? if not then based on the screenshot that is a new callsign for the band/mode combination and how it's designed to work.

**Comment by ryandzink** [comment]
@magicbug maybe I am misunderstanding what criteria cloudlog is taking into account for considering something a new callsign. For satellites is it that you have never worked that contact on that satellite?

I just tested this out with K8YSE/7 above, who I have made contact with on AO-27 in the past, and even after choosing that satellite on the Satellite tab he still shows up as a new callsign with a red border, even with that previous AO-27 contact displaying under the map.
